app/assets/stylesheets/buildout_design_system/components/badge.scss

```diff
@@ -59,6 +48,6 @@
 
   &.badge-xs {
     @include rfs(0.75rem, --#{$prefix}badge-font-size);
-    --#{$prefix}badge-padding-y: 0.125rem;
+    --#{$prefix}badge-padding-y: 0.0625rem;
```

Comment:
if we add `box-sizing: border-box` we should be able to use the regular padding as they would then start including the border and padding all in one

Reply:
hmm this didnt seem to work for me!! it looks like all elements already have box`-sizing: border-box` applied to them